README.md

```diff
@@ -34,7 +34,7 @@ yarn add @mongodb-js/charts-embed-dom
 
 - [Getting started with the MongoDB Charts SDK](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/charts/unauthenticated)
 - [Getting started with custom JWT authentication](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/charts/authenticated-custom-jwt)
-- [Getting started with Realm authentication](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/charts/authenticated-realm)
+- [Getting started with Realm Web authentication](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/charts/authenticated-realm-web)
```

Comment:
I wonder to what extent we would like to replace realm web with `App Services`, I know the npm package name is called `realm-web`, should we keep it just at npm level and refer everything else as App services?

Reply:
Tricky question 😬 Would it be ok if we have it with both names here as this `readme` is the gateway to all examples and is the thing which everyone sees when they come to the repo. Everywhere else, we can refer to it as `App Services`. So `Getting started with App Services (Realm Web) authentication`. What do you think?

Reply:
> How about `"Getting started with App Services authentication via Realm Web SDK"`?

This sounds like a good idea 👍